src/main/java/com/google/opensesame/servlets/MentorsServlet.java

```diff
@@ -26,7 +26,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
-    addMockMentor(response); // TODO: remove by production
+    // addMockMentor(response); // TODO: remove by production
```

Comment:
Yes! The addMockMentor function only has to be called once to add a mock mentor to datastore. It is commented out here so that people running their own dev servers that wish to add the mock mentor can choose to, but don't get it automatically.

Reply:
nit: would rather remove it than comment it out, just to keep things clean.